am-ui/src/app/components/profile/profile.component.html

```diff
@@ -40,8 +40,47 @@ <h1>
         >
       </div>
     </div>
-    <div class="result-banner" *ngIf="saveAction" [ngClass]="{error: saveError}">
-      <p *ngIf="saveAction && !saveError">{{ 'profile.success' | translate }}</p>
+
+    <div class="flex-container-wrapper">
+      <div>
+        <button
+          *ngIf="!showModal"
+          type="button"
+          class="btn btn-primary"
+          (click)="showDeleteProfileModal()"
+        >
+          {{ 'profile.deleteProfile' | translate }}
+        </button>
+      </div>
+      <div>
+        <mp-custom-modal
+          [isModalVisible]="showModal"
+          modalTitleI18n="profile.modal.title"
+          modalBodyI18n="profile.modal.body"
+          modalPrimaryButtonI18n="ad.delete"
+          (modalCloseEvent)="hideDeleteProfileModal()"
+          (modalSubmitEvent)="onDeleteProfile()"
+        >
+        </mp-custom-modal>
+      </div>
+    </div>
+
+    <div
+      *ngIf="showDeleteUserAlert"
+      class="result-banner"
+      [ngClass]="deleteUserAlertCssClassesArray"
+    >
+      {{ deleteUserAlertText }}

```

Comment:
Is there a | translate missing?

Reply:
Translate is implemented in the ocde 
https://github.com/AkrosAG/Akros-Marketplace/pull/277/files#r1063529842